### ISSUE TITLE
Flaky TestDeltaTaskFailureRecoveryTest.testUpdateWithSubquery

### ISSUE THREAD

**Issue by ebyhr** [trinodb/trino#12268]
```
Error:  Tests run: 19, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 1,640.962 s <<< FAILURE! - in io.trino.plugin.deltalake.TestDeltaTaskFailureRecoveryTest
Error:  io.trino.plugin.deltalake.TestDeltaTaskFailureRecoveryTest.testUpdateWithSubquery  Time elapsed: 89.728 s  <<< FAILURE!
java.lang.AssertionError: 

Expecting throwable message:
  <"Error closing remote buffer (http://127.0.0.1:41977/v1/task/20220506_020701_01464_rmmkb.1.0.0/results/0 - 3 failures, failure duration 10.05s, total failed request time 15.06s)">
to contain:
  <"Encountered too many errors talking to a worker node">
but did not.

Throwable that failed the check:

java.lang.RuntimeException: Error closing remote buffer (http://127.0.0.1:41977/v1/task/20220506_020701_01464_rmmkb.1.0.0/results/0 - 3 failures, failure duration 10.05s, total failed request time 15.06s)
	at io.trino.testing.AbstractTestingTrinoClient.execute(AbstractTestingTrinoClient.java:122)
	at io.trino.testing.DistributedQueryRunner.executeWithQueryId(DistributedQueryRunner.java:541)
	at io.trino.testing.BaseFailureRecoveryTest$FailureRecoveryAssert.execute(BaseFailureRecoveryTest.java:565)
	at io.trino.testing.BaseFailureRecoveryTest$FailureRecoveryAssert.executeActual(BaseFailureRecoveryTest.java:551)
	at io.trino.testing.BaseFailureRecoveryTest$FailureRecoveryAssert.executeActualNoRetries(BaseFailureRecoveryTest.java:536)
	at io.trino.testing.BaseFailureRecoveryTest$FailureRecoveryAssert.lambda$failsWithoutRetries$8(BaseFailureRecoveryTest.java:667)
	at org.assertj.core.api.ThrowableAssert.catchThrowable(ThrowableAssert.java:62)
	at org.assertj.core.api.AssertionsForClassTypes.catchThrowable(AssertionsForClassTypes.java:877)
	at org.assertj.core.api.Assertions.catchThrowable(Assertions.java:1306)
	at org.assertj.core.api.Assertions.assertThatThrownBy(Assertions.java:1178)
	at io.trino.testing.BaseFailureRecoveryTest$FailureRecoveryAssert.failsWithoutRetries(BaseFailureRecoveryTest.java:667)
	at io.trino.testing.BaseFailureRecoveryTest.testNonSelect(BaseFailureRecoveryTest.java:457)
	at io.trino.testing.BaseFailureRecoveryTest.testTableModification(BaseFailureRecoveryTest.java:368)
	at io.trino.testing.BaseFailureRecoveryTest.testTableModification(BaseFailureRecoveryTest.java:363)
	at io.trino.testing.BaseFailureRecoveryTest.testUpdateWithSubquery(BaseFailureRecoveryTest.java:296)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: io.trino.spi.TrinoTransportException: Error closing remote buffer (http://127.0.0.1:41977/v1/task/20220506_020701_01464_rmmkb.1.0.0/results/0 - 3 failures, failure duration 10.05s, total failed request time 15.06s)
	at io.trino.operator.HttpPageBufferClient$2.onFailure(HttpPageBufferClient.java:531)
	at com.google.common.util.concurrent.Futures$CallbackListener.run(Futures.java:1132)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: java.lang.RuntimeException: java.util.concurrent.TimeoutException: Idle timeout 5000 ms
	at io.airlift.http.client.ResponseHandlerUtils.propagate(ResponseHandlerUtils.java:25)
	at io.airlift.http.client.StatusResponseHandler.handleException(StatusResponseHandler.java:45)
	at io.airlift.http.client.StatusResponseHandler.handleException(StatusResponseHandler.java:28)
	at io.airlift.http.client.jetty.JettyResponseFuture.failed(JettyResponseFuture.java:120)
	at io.airlift.http.client.jetty.BufferingResponseListener.onComplete(BufferingResponseListener.java:85)
	at org.eclipse.jetty.client.ResponseNotifier.notifyComplete(ResponseNotifier.java:218)
	at org.eclipse.jetty.client.ResponseNotifier.notifyComplete(ResponseNotifier.java:210)
	at org.eclipse.jetty.client.HttpReceiver.terminateResponse(HttpReceiver.java:481)
	at org.eclipse.jetty.client.HttpReceiver.terminateResponse(HttpReceiver.java:461)
	at org.eclipse.jetty.client.HttpReceiver.abort(HttpReceiver.java:557)
	at org.eclipse.jetty.client.HttpChannel.abortResponse(HttpChannel.java:152)
	at org.eclipse.jetty.client.HttpChannel.abort(HttpChannel.java:145)
	at org.eclipse.jetty.client.HttpExchange.abort(HttpExchange.java:264)
	at org.eclipse.jetty.client.HttpConversation.abort(HttpConversation.java:164)
	at org.eclipse.jetty.client.HttpRequest.abort(HttpRequest.java:819)
	at org.eclipse.jetty.client.http.HttpConnectionOverHTTP.abort(HttpConnectionOverHTTP.java:214)
	at org.eclipse.jetty.client.http.HttpConnectionOverHTTP.close(HttpConnectionOverHTTP.java:200)
	at org.eclipse.jetty.client.http.HttpConnectionOverHTTP.onIdleExpired(HttpConnectionOverHTTP.java:160)
	at org.eclipse.jetty.io.AbstractEndPoint.onIdleExpired(AbstractEndPoint.java:402)
	at org.eclipse.jetty.io.IdleTimeout.checkIdleTimeout(IdleTimeout.java:171)
	at org.eclipse.jetty.io.IdleTimeout.idleCheck(IdleTimeout.java:113)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
	... 3 more
Caused by: java.util.concurrent.TimeoutException: Idle timeout 5000 ms
	... 10 more

	at io.trino.testing.BaseFailureRecoveryTest.lambda$testNonSelect$22(BaseFailureRecoveryTest.java:457)
	at io.trino.testing.BaseFailureRecoveryTest$FailureRecoveryAssert.failsWithoutRetries(BaseFailureRecoveryTest.java:667)
	at io.trino.testing.BaseFailureRecoveryTest.testNonSelect(BaseFailureRecoveryTest.java:457)
	at io.trino.testing.BaseFailureRecoveryTest.testTableModification(BaseFailureRecoveryTest.java:368)
	at io.trino.testing.BaseFailureRecoveryTest.testTableModification(BaseFailureRecoveryTest.java:363)
	at io.trino.testing.BaseFailureRecoveryTest.testUpdateWithSubquery(BaseFailureRecoveryTest.java:296)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:829)
```

https://github.com/trinodb/trino/runs/6316240517

**Comment by ebyhr** [comment]
cc: @arhimondr @losipiuk

**Comment by losipiuk** [comment]
duplicate of https://github.com/trinodb/trino/issues/12151

**Comment by ebyhr** [comment]
@losipiuk Could you fix the title of #12151?
